### DISCUSSION TITLE
transmission capacity init AC ba NARIS2024

### DISCUSSION THREAD

**Discussion by chrigraf** [NREL/ReEDS-2.0#174] [Q&A: Data] [answered]
I'd have a question regarding ./inputs/transmission/transmission_capacity_init_AC_ba_NARIS2024.csv: The interface p119||p122 has values for MW_f0: 972.166 and MW_r0: 171.377. Is it correct to say that max AC export from p119 is 972.166 MW and max AC import to p119 is 171.377 MW, i.e., -171.377 <= flow(p119->p122) <= 972.166? The reason i am asking is because the existing generation and storage capacity (./capacitydata/ReEDS_generator_database_final_EIA-NEMS.csv), in p119 is quite low (193.1 MW but mostly onshore wind). 
Thank you!

**Comment by wesleyjcole** [comment]
Yes, your understanding is correct.  The transmission capacity limits are calculated using maximum capacity flows rather than existing capacity.  See https://arxiv.org/abs/2308.03612 for more information on how these limits were calculated.

**Comment by chrigraf** [comment]
Thank you. Appreciate it!
Given the low import capacity, the low generation and storage capacity in p119 (193.1 MW but mostly onshore wind), and the relatively high load >400 MW (on average, using historic data) this means load will be curtailed in most hours in this zone? Just wanted to make sure that the import/export capacity should not be reversed. Thank you!

**Comment by wesleyjcole** [comment, marked as answer]
Good question.  I just pulled a recent ReEDS run and saw that p119 does have a lot of dropped load until the model is allowed to build new capacity.  I'll create an issue on our internal repository and we'll see if we can figure out what's going on with this issue (e.g., is the issue with transmission, load, or generation capacity) and push out a fix in a future version.

**Comment by chrigraf** [comment]
Thanks you very much. Appreciate the great work you all are doing!
